package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "jest-preset-angular",
-  "version": "13.0.1",
+  "version": "13.0.2",
```

Comment:
I believe @ahnpnl bumps the version at release time.  Not sure if it's appropriate to include it in this PR

Reply:
yes the version is bump during the release process

Reply:
Reset the version